src/bench/coin_selection.cpp

```diff
@@ -56,10 +56,11 @@ static void CoinSelection(benchmark::Bench& bench)
     addCoin(3 * COIN, wallet, wtxs);
 
     // Create coins
+    CCoinControl coin_control:
```

Comment:
```suggestion
    wallet::CCoinControl coin_control;
```

Reply:
Or pass a nullptr instead of an empty `CCoinControl`.

Reply:
Done. Replaced with nullptr